src/main/java/com/gmail/yuramitryahin/model/Role.java

```diff
@@ -0,0 +1,23 @@
+package com.gmail.yuramitryahin.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "role")
+public class Role {
+    @Id
+    @Enumerated(value = EnumType.STRING)
```

Comment:
let's have Long id as a PK, otherwise it'll be more complicated to update role name in future

Reply:
fixed